Card.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+
+public class Card
+{
+    public enum Color{
+        Red, Black
+    }
+
+    public enum Suit{
+        Hearts, Diamonds, Spades, Clubs
+    }
+
+    public int value;
```

Comment:
Class fields should be private w/ getters (don't add setters since the class should be immutable)